src/js/index.js

```diff
@@ -111,8 +118,11 @@ async function init() {
     pc.config.api = api;
     pc.profile = data.id;
     pc.config.baseUrl = pc.baseUrl;
-    // TODO add this to config - check the <script> tag in the HTML which hardcodes this value
-    pc.config.analytics = new Analytics('UA-93649482-25', pc.profile);
+    if (GOOGLE_ANALYTICS_ID !== "undefined" && GOOGLE_ANALYTICS_ID !== "") {
```

Comment:
"undefined" in quotes could be what's caused analytics to show up where it shouldn't

Reply:
@michaelglenister "undefined" in quotes was not the problem but looks like in index.html there is some google analytics related script hardcoded

![image](https://user-images.githubusercontent.com/53019884/163331183-be8e8984-2bec-4e76-a707-53a3812c27e1.png)

Reply:
but since we don't init Analytics class, the events will not be caught and they will not be logged to GA